packages/amplify_api_rest/pubspec.yaml

```diff
@@ -0,0 +1,70 @@
+name: amplify_api_rest
+description: A new flutter plugin project.
+version: 0.0.1
+author:
+homepage:
+
+environment:
+  sdk: ">=2.7.0 <3.0.0"
+  flutter: ">=1.20.0 <2.0.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+  plugin_platform_interface: ^1.0.1
+
+  amplify_api_rest_plugin_interface:
+    path: ../amplify_api_rest_plugin_interface
```

Comment:
With melos, local paths are no longer needed - as long as the specified version exists locally.